CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## Changed
+
+- `go.sum` parser to filter out modules with /go.mod
```

Comment:
I think a more user-focused message would be appreciated.
This goes into details on what we do, but I think users mainly care about the parser not spitting out packages that aren't used anymore.